src/Agent.Listener/_project.json

```diff
@@ -33,7 +33,7 @@
     "dependencies": {
         "NETStandard.Library": "1.0.0-rc3-23829",
         "Microsoft.VisualStudio.Services.Agent": { "target": "project", "version": "1.0.0-*" },
-        "vss-api-netcore": "0.5.4-private",
+        "vss-api-netcore": "0.5.5-private",
```

Comment:
is the version change intentional? are we pulling in dependency on 0.5.5 or is it used in agent already?